src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Requests/DataSetQuerySort.cs

```diff
@@ -6,11 +7,13 @@ namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Api.Requests;
 public record DataSetQuerySort
 {
     /// <summary>
-    /// The name of the field to sort by. This can be one of:
+    /// The name of the field to sort by. This can be:
     ///
-    /// - `TimePeriod`
-    /// - A geographic level code e.g. `NAT`, `LA`
-    /// - A filter ID e.g. `characteristic`
+    /// - `time_period`
+    /// - `geographic_level`
+    /// - A location level code e.g. `NAT` `REG`, `LA`
```

Comment:
what's the difference between 
- `geographic_level`
- A location level code e.g. `NAT` `REG`, `LA`

Reply:
I understand this now :) 

They are indeed different